mne/decoding/transformer.py

```diff
@@ -328,22 +338,16 @@ def transform(self, epochs_data, y=None):
         if not isinstance(epochs_data, np.ndarray):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
-
-        epochs_data = np.atleast_3d(epochs_data)
-
-        n_epochs, n_channels, n_times = epochs_data.shape
-        X = epochs_data.reshape(n_epochs * n_channels, n_times)
-
-        psd, _ = multitaper_psd(x=X, sfreq=self.sfreq, fmin=self.fmin,
-                                fmax=self.fmax, bandwidth=self.bandwidth,
-                                adaptive=self.adaptive, low_bias=self.low_bias,
-                                n_jobs=self.n_jobs,
-                                normalization=self.normalization,
-                                verbose=self.verbose)
-
-        _, n_freqs = psd.shape
-        psd = psd.reshape(n_epochs, n_channels, n_freqs)
-
+        psd, freqs = compute_epochs_psd(epochs_data, Fs=self.sfreq,
+                                        fmin=self.fmin, fmax=self.fmax,
+                                        mt_bandwidth=self.bandwidth,
+                                        mt_adaptive=self.adaptive,
+                                        mt_low_bias=self.low_bias,
+                                        mt_normalization=self.normalization,
+                                        n_jobs=self.n_jobs,
+                                        verbose=self.verbose)
+
```

Comment:
good idea - will do. I can explain in the docstring what parameters are used for what method.